### DISCUSSION TITLE
Tiny inconsistency in Tutorial 2

### DISCUSSION THREAD

**Discussion by seanshahkarami** [encode/django-rest-framework#8573] [Ideas & Suggestions]
Hello everyone! Thank you so much for the great work and tutorials!

I just wanted to point out a _very_ small inconsistency in [Tutorial 2: Requests and Responses](https://www.django-rest-framework.org/tutorial/2-requests-and-responses/).

That tutorial uses the following example route:

```python
    # no trailing slash
    path('snippets/<int:pk>', views.snippet_detail),
```

whereas all the other examples, before and after, use the trailing slash:

```python
    # has trailing slash
    path('snippets/<int:pk>/', views.snippet_detail),
```

**Comment by 2ykwang** [comment]
thank you I submitted a PR #8604
